Looks like you have compilation issues https://www.travis-ci.org/WCSim/WCSim/builds/332080271

src/WCSimDetectorConstruction.cc: In member function 'void WCSimDetectorConstruction::CreateCombinedPMTQE(std::vector<G4String, std::allocator<G4String> >)':
src/WCSimDetectorConstruction.cc:284: warning: '>>' operator will be treated as two right angle brackets in C++0x
src/WCSimDetectorConstruction.cc:284: warning: suggest parentheses around '>>' expression
src/WCSimDetectorConstruction.cc:284: error: 'QEMap' was not declared in this scope
src/WCSimDetectorConstruction.cc:284: error: '>>' should be '> >' within a nested template argument list
/root/HyperK/Geant4-10.1.3-Linux/include/Geant4/CLHEP/Units/PhysicalConstants.h: At global scope:
/root/HyperK/Geant4-10.1.3-Linux/include/Geant4/CLHEP/Units/PhysicalConstants.h:71: warning: 'CLHEP::electron_charge' defined but not used
make: *** [/root/HyperK/WCSim/exe/tmp/Linux-g++/WCSim/WCSimDetectorConstruction.o] Error 1

--- a/src/WCSimDetectorConstruction.cc
+++ b/src/WCSimDetectorConstruction.cc
@@ -281,7 +281,7 @@ void WCSimDetectorConstruction::CreateCombinedPMTQE(std::vector<G4String> Collec
 
   // Define relevant variable
   // Create array of maps for CollectionName
-  std::vector<std::map<G4float,G4float>> QEMap;
+  std::vector< std::map<G4float,G4float> > QEMap;
   std::vector<G4float> maxQEVec;
   // Need size of QE array
   std::vector<G4int> NbOfWLBins;

